core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -251,6 +252,7 @@ object KafkaConfig {
   val ConsumerGroupMaxHeartbeatIntervalMsProp ="group.consumer.max.heartbeat.interval.ms"
   val ConsumerGroupMaxSizeProp = "group.consumer.max.size"
   val ConsumerGroupAssignorsProp = "group.consumer.assignors"
+  val GroupConsumerUpgradePolicyProp = "group.consumer.upgrade.policy"
```

Comment:
I am still not fully satisfied with this property because `group.consumer.upgrade.policy = upgrade` reads weird. What could we use to replace `upgrade` here? `conversion`? `migration`?

Reply:
nit: `ConsumerGroupUpgradePolicyProp`

Reply:
Let me change it to `consumer.group.migration.policy`